packages/app/src/runner/SpecRunner.vue

```diff
@@ -3,68 +3,84 @@
     id="main-pane"
     class="flex border-gray-900 border-l-1"
   >
-    <HideDuringScreenshot
-      id="inline-spec-list"
-      class="bg-gray-1000"
+    <ResizablePanels
+      :offset-left="64"
+      :max-total-width="windowWidth - 64"
+      :initial-panel1-width="initialSpecsListWidth"
+      :initial-panel2-width="initialReporterWidth"
+      :show-panel1="runnerUiStore.isSpecsListOpen && !screenshotStore.isScreenshotting"
+      :show-panel2="!screenshotStore.isScreenshotting"
+      @resize-end="handleResizeEnd"
+      @panel-width-updated="handlePanelWidthUpdated"
     >
+      <template #panel1="{isDragging}">
+        <HideDuringScreenshot
+          id="inline-spec-list"
+          class="h-full bg-gray-1000"
+          :class="{'pointer-events-none': isDragging}"
+        >
+          <div
```

Comment:
Should these v-if and `v-show be on the `<HideDuringScreenshot>`? Does the additional `div` do anything here?

Reply:
Good call, works fine without the extra div